lib/exporters/default.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Exporters
+  class Default
+    attr_reader :collection
+
+    def initialize(collection = [])
+      @collection = collection
+    end
+
+    def generate
+      raise NotImplementedError
+    end
+
+    class << self
+      def content(collection)
+        new(collection).generate
+      end
+
+      def filename
+        'export.csv'
```

Comment:
this should be custom right?

Reply:
It's true but on this case is a default method for exporter class in a case of a children subclass not implement it.

https://github.com/Codeminer42/cm42-central/pull/528/files#diff-a41bca78f4a0a5f0ff53fa760b658590R14